src/nvim/window.c

```diff
@@ -1922,9 +1922,7 @@ static void win_rotate(bool upwards, int count)
   redraw_all_later(NOT_VALID);
 }
 
-/*
- * Move the current window to the very top/bottom/left/right of the screen.
- */
+// Move window "wp" to the very top/bottom/left/right of the screen.
 static void win_totop(int size, int flags)
```

Comment:
there's no `wp` param, is there a change planned here?

Reply:
Initialize I had to change the function when `vim.cmd_preview` was still a thing in the PR, after reverting those changes I probably forgot to change the comments, will fix it